en/advanced_config/esc_calibration.md

```diff
@@ -24,33 +38,42 @@ To calibrate the ESCs:
 1. Remove the propellers.
 
    :::warning
-   Never attempt ESC calibration with props on.
-   
+   Never attempt ESC calibration with propellers on!
+
    The motors should not spin during ESC calibration.
-   However if an ESC doesn't properly support/detect the calibration sequence then it will respond to the PWM input by running the motor at maximum speed.
+   However if the calibration starts when the ESC is powered already or the ESC doesn't properly support/detect the calibration sequence then it will respond to the PWM input by running the motor at maximum speed.

```

Comment:
```suggestion
   However if calibration starts when the ESC is powered already, or if the ESC doesn't properly support/detect the calibration sequence, then it will respond to the PWM input by running the motor at maximum speed.

```